### PR TITLE
Add nft types

### PR DESCRIPTION
**Pull request by matzapatasphereone**

### CODE REVIEW

File: src/types.ts [hunk 1]
```diff
@@ -120,6 +120,9 @@ export interface ChargeItem {
   image: string;
   amount: number;
   quantity: number;
+  nftUri: string;
```
Comment (davidw-sphereone) [inline src/types.ts:123]: These should be optional params

**Comment by davidw-sphereone** [comment]
LGTM

**Comment by davidw-sphereone** [comment]
I have updated the UnitySDK to reflect this change also

**Review by DiegoCerdaC — APPROVED** [review]
Looks good to me !

**Pull request merged**